utils/lists.py

```diff
@@ -1,6 +1,15 @@
+from utils.trivia_data import QuestionAnswer, QuestionList
+
 ballresponse = [
-  'Yes', 'No', 'Take a wild guess...', 'Very doubtful',
-  'Sure', 'Without a doubt', 'Most likely', 'Might be possible',
-  "You'll be the judge", 'no... (╯°□°）╯︵ ┻━┻', 'no... baka',
-  'senpai, pls no ;-;'
+    'Yes', 'No', 'Take a wild guess...', 'Very doubtful',
```

Comment:
I'm not sure of the reason for the ball response code... Was the Ball response code changed to be for the trivia?

Reply:
This is pre-existing code used with the `!8ball`/`!eightball` command. The only reason I touched it is because my automatic linter in VS Code was screaming at me about under-indentation (because 4 spaces is a near-universal standard in Python), and in the middle of frustration of trying to fix a different, more relevant problem, changing the amount of indentation was a faster way to shut it up than searching for the hidden option to silence that alert.